R/financialyear.R

```diff
@@ -0,0 +1,28 @@
+#' @title Assign date to a financial year
+#'
+#' @description \code{fin_year} takes a date and assigns it to the correct
+#' financial year in the PHI specified format.
+#'
+#' @details The PHI accepted format for financial year is yyyy/yy e.g. 2017/18.
+#'
+#' @param date A date which must be supplied in standard date format. The
+#' function as.Date() or dmy (from lubridate) can be used to change a variable
+#' to date format.
```

Comment:
I think it's good to offer some suggestions on how to give a variable `Date` class, but would it be okay to make it clear that these are just a couple of the functions which can achieve that, and not the only ones. Also, would it be okay to use the word 'class' instead of format? Just to make it clear that entering something which is technically in date format but isn't recognised by R as a date (like a character variable in the format `YYYY-MM-DD`) won't work here.